src/states_screens/race_gui_base.cpp

```diff
@@ -1039,6 +1039,21 @@ void RaceGUIBase::drawPlayerIcon(AbstractKart *kart, int x, int y, int w)
                                                       NULL, true);
         }
     }
+
+	//lap flag for finished karts
+	if (kart->hasFinishedRace())
+	{
+        video::ITexture *icon_lapflag = irr_driver->getTexture(FileManager::GUI_ICON, "lap_flag.png");
+        if (icon_lapflag != NULL)
+		{
```

Comment:
our conventions are to indent using spaces, not tabs

Reply:
Sorry, I thought I converted everything to space, seems like I missed that line